x-pack/plugins/triggers_actions_ui/kibana.jsonc

```diff
@@ -25,7 +26,9 @@
       "dashboard",
       "licensing",
       "expressions",
-      "lens"
+      "lens",
+      "controls",
+      "embeddable"
```

Comment:
Are controls and embeddable used somehwere in this plugin? I might just be missing it!

Reply:
They're necessary in order to pass the controls embeddable renderer to the `AlertFilterControls` component. This happens in the Stack (unified) alerts page which is part of the `triggers_actions_ui` plugin 🙂